exp-player/addon/components/exp-lookit-dialogue-page/template.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="exp-lookit-dialogue-page" id="exp-lookit-dialogue-page-container">
```

Comment:
Good idea! I've shared such a directory with you and Sara. I'll follow up next week with more details about planning for while I'm on leave - we may be able to avoid having them ask questions about general frame usage, and let you have some less-interrupted time to focus on new features :)